po/el.po

```diff
@@ -1,506 +1,1370 @@
-# PDF-Shuffler translation files.
-# Copyright (C) 2008-2017 Konstantinos Poulios
-# This file is part of PDF-Shuffler which is released under the GNU General
+# PDF Arranger translation files.
+#
+# This file is part of PDF Arranger which is released under the GNU General
 # Public License version 3 or later.
 # See file COPYING or go to <http://www.gnu.org/licenses/> for full license details.
+#
 # Translators:
 # Konstantinos Poulios, 2008-2017
+# Manussos Pulinakis, 2022
+#
 msgid ""
 msgstr ""
-"Project-Id-Version: PDF-Shuffler 0.7\n"
+"Project-Id-Version: PDF Arranger 1.9.0\n"
+<<<<<<< Updated upstream
+<<<<<<< Updated upstream
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2021-01-01 22:06+0100\n"
-"PO-Revision-Date: 2017-02-26 16:16+0000\n"
-"Last-Translator: Konstantinos Poulios <logari81@gmail.com>\n"
-"Language-Team: Greek (http://www.transifex.com/logari81/pdfshuffler/language/"
-"el/)\n"
+"POT-Creation-Date: 2022-05-18 09:23+0300\n"
+=======
+<<<<<<< HEAD
+"POT-Creation-Date: \n"
+=======
+=======
+<<<<<<< HEAD
+"POT-Creation-Date: \n"
+=======
+<<<<<<< Updated upstream
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-05-18 09:23+0300\n"
+=======
+<<<<<<< HEAD
+"POT-Creation-Date: \n"
+=======
+>>>>>>> Stashed changes
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2022-05-18 09:23+0300\n"
+>>>>>>> 7145288 (Update the Greek translation)
+>>>>>>> Stashed changes
+<<<<<<< Updated upstream
+=======
+>>>>>>> d767cd6 (Update the Greek translation)
+>>>>>>> Stashed changes
+"PO-Revision-Date: \n"
+"Last-Translator: Manussos Pulinakis <manussos@protonmail.com>\n"
+"Language-Team: \n"
 "Language: el\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
+"X-Generator: Poedit 3.0.1\n"
 
-#: pdfarranger/core.py:80
+#: pdfarranger/core.py:87
 msgid "page"
 msgstr "σελίδα"
 
-#: pdfarranger/core.py:175
+#: pdfarranger/core.py:191
 msgid "Password required"
-msgstr ""
+msgstr "Απαιτείται κωδικός"
 
-#: pdfarranger/core.py:186
+#: pdfarranger/core.py:202
 msgid "The password will be remembered until you close PDF Arranger."
-msgstr ""
-
-#: pdfarranger/core.py:187
+<<<<<<< Updated upstream
```

Comment:
These are leftovers from a git merge/rebase conflict. Those need to be cleaned up before we can move on with this.

Reply:
This should also fix CI. I'm not sure how you ended up with this situation, tbh, but I know Git can be confusing at the beginning. Let me know if you need any assistance :wink:

Reply:
Yeah, working with git is a joyful experience 😵‍💫
No idea how I got it messed up like that!

Reply:
Should be okay now, hopefully 🤞...

Reply:
It becomes joyful (or at least useful) when you get used to Git :wink: